pages/pil/dashboard/index.js

```diff
@@ -39,6 +47,16 @@ module.exports = settings => {
   app.use((req, res, next) => {
     const values = get(req.session, `form[${req.model.id}].values`);
     req.model = { ...req.model, ...values };
+
+    const establishmentTransfer = req.user.profile.establishments
+      .filter(e => e.id !== req.establishment.id)
+      .find(e => e.id === get(req.model, 'establishmentId'));
+
+    req.model.establishment = {
+      from: pick(req.establishment, ['id', 'name']),
```

Comment:
This feels like it should be taken from the PIL model, req.pil.establishment - I know with current business logic it does also happen to be the establishment you are scoped to in the URL but if that changes in future, or if we allow someone with perms at 2 establishments to view this pil then this logic becomes broken. I'd personally eager load the establishment at API level